app/scripts/lib/metrics.js

```diff
@@ -503,23 +547,6 @@ define(function (require, exports, module) {
       return this._isSampledUser;
     },
 
-    logFlowBegin (flowId, flowBeginTime) {
-      // Don't emit a new flow.begin event unless flowId has changed.
-      if (flowId !== this._flowId) {
-        this._flowId = flowId;
-        this._flowBeginTime = flowBeginTime;
-        this.logFlowEvent('begin');
-      }
-    },
-
-    logFlowEvent (eventName, viewName) {
-      this.logEvent(marshallFlowEvent(eventName, viewName));
-    },
-
-    logFlowEventOnce (eventName, viewName) {
-      this.logEventOnce(marshallFlowEvent(eventName, viewName));
-    },
-
     getFlowEventMetadata () {
       const metadata = (this._flowModel && this._flowModel.attributes) || {};
```

Comment:
I realize this was already here, but it seems like this could be simplified to:

```js
  return this._flowModel && this._flowModel.pick('flowBegin', 'flowId');
```

And then where [the data is consumed](https://github.com/mozilla/fxa-content-server/pull/4681/files#diff-7b6eb0846bca35324d8b1acd38a32302R295), you'd translate `flowBegin` to `flowBeginTime`